libr/core/cmd_anal.c

```diff
@@ -6915,18 +6919,23 @@ static void cmd_anal_aav(RCore *core, const char *input) {
 			//TODO: Reduce multiple hits for same addr
 			from = r_itv_begin (map2->itv);
 			to = r_itv_end (map2->itv);
-			eprintf ("Value from 0x%08"PFMT64x " to 0x%08" PFMT64x "\n", from, to);
+			snprintf (out, sizeof(out), "Value from 0x%08"PFMT64x " to 0x%08" PFMT64x " (aav)", from, to);
+			rowlog (core, out);
+			rowlog_done (core);
 			r_list_foreach (list, iter, map) {
 				ut64 begin = map->itv.addr;
 				ut64 end = r_itv_end (map->itv);
 				if (r_cons_is_breaked ()) {
 					break;
 				}
 				if (end - begin > UT32_MAX) {
-					eprintf ("Skipping huge range\n");
+					rowlog (core, "Skipping huge range");
+					rowlog_done (core);
 					continue;
 				}
-				eprintf ("aav: 0x%08"PFMT64x"-0x%08"PFMT64x" in 0x%"PFMT64x"-0x%"PFMT64x"\n", from, to, begin, end);
+				snprintf (out, sizeof(out), "0x%08"PFMT64x"-0x%08"PFMT64x" in 0x%"PFMT64x"-0x%"PFMT64x" (aav)", from, to, begin, end);
```

Comment:
what about making rowlogf and get the format string handling directly inside the function instead of using snprintf all the time

Reply:
I checked the define of eprintf but it doesn't seem to help, do you mind creating this helper? I will change accordingly afterwards?